### DISCUSSION TITLE
Need new function: Quickly switch audio devices

### DISCUSSION THREAD

**Discussion by yujiadong24** [File-New-Project/EarTrumpet#1100] [Ideas]
_I'm using translator._

Want to add a function in the shortcut settings: quick switch audio devices, for different devices bound to different shortcut keys.

**Comment by riverar** [comment]
Hi @yujiadong24, try our experimental builds (https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller). This includes EarTrumpet Actions which will let you do this.

![image](https://user-images.githubusercontent.com/475132/159523562-0c93ddfe-ba54-4e65-ba55-2b5f5675dec1.png)
